chainer/functions/array/resize_images.py

```diff
@@ -288,10 +306,13 @@ def resize_images(x, output_shape):
         output_shape (tuple): This is a tuple of length 2 whose values are
             :obj:`(h_O, w_O)`. Note that the order of height and width is
             opposite of the one in OpenCV.
+        align_corners (bool): When this value is :obj:`True`,
+            the corners of the input are mapped to the corners of
+            the output.
 
```

Comment:
Could you provide more information on the algorithm used to do this?
OpenCV is using a bilinear interpolation method by default to keep the corners in place and fill the space.
But it will be great if we can add this to the documentation.

Reply:
How much level of detail do you want the documentation to have?
If necessary, I can write a formula of how the mapping between the input and the output happens.

This option is quite common in other DL frameworks, so the simple docs may be adequate.
https://www.tensorflow.org/api_docs/python/tf/image/resize_bilinear

Reply:
Just by adding a reference or a high-level general description is fine :D.

Reply:
I think the best way to illustrate this option briefly is by adding a code example. I will work on that.

Reply:
Thanks, but you don't need to go that far! :)
Just a single sentence saying that the default behavior is the bilinear interpolation as in opencv should suffice.

Reply:
OK!
I added a sentence.